### ISSUE TITLE
Microsoft 365 community feedback - 

### ISSUE THREAD

**Issue by ddaanniieell9** [MicrosoftDocs/microsoft-365-community#956]
### Type of issue

Other (describe below)

### Feedback

Was this note intended as a joke? If not, it doesn't make sense.

https://learn.microsoft.com/en-us/microsoft-365/community/versioning-basics-best-practices#disabling-versioning
"A Cautionary Tale: As site owner, if you disable Versioning and don't tell your end users, they'll notify you. In person."

### Page URL

https://learn.microsoft.com/en-us/microsoft-365/community/versioning-basics-best-practices#disabling-versioning

### Content source URL

https://github.com/MicrosoftDocs/microsoft-365-community/blob/main/Community/versioning-basics-best-practices.md

### Author

@PatD

### Document Id

c06f1880-827d-1d0b-c0fc-601432749411

**Comment by sympmarc** [comment]
Yes, that's a tongue-in-cheek joke, however valid it is. In the Community Docs, we sometimes take a bit more license than Microsoft does in their official documents.

Do you think the line detracts from the value of the article?

**Comment by eemancini** [comment]
Updated article
